internal/datasource/config/config_writer_test.go

```diff
@@ -25,10 +27,19 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const (
+	instanceID    = "aecea348-62c1-4e3d-b848-6d6cdeb1cb9c"
```

Comment:
can you get the instance Id from the test helpers?

Reply:
Ya, it's not being used was left in by accident